### PR TITLE
RYA-448 Implement JAXB marshalling code for the Query Manager's XML c…

### PR DESCRIPTION
**Pull request by kchilton2**
Description
Implemented configuration XML unmarshalling code and a template xml file.

Tests
Added unit tests.

Links
https://issues.apache.org/jira/browse/RYA-443

### CODE REVIEW

File: extras/rya.streams/query-manager/src/main/config/configuration.xml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    <performanceTunning>
+        <!-- Indicates how frequently the Query Change Logs that are defined
+           - in the sources section of this configuration file are polled for
+           - updates. This value effects how much time may pass between the 
+           - state of a query changing and that change being represented by
+           - the execution engine.
+           -->
+        <queryChanngeLogDiscoveryPeriod>
+            <value>1</value>
+            <!-- Options: {MILLISECONDS, SECONDS, MINUTES} -->
```
Comment (isper3at) [inline extras/rya.streams/query-manager/src/main/config/configuration.xml:47]: There are a lot more TimeUnit options

Comment (kchilton2) [reply extras/rya.streams/query-manager/src/main/config/configuration.xml:47]: I don't think it makes sense to use any of the other options, so I'm choosing to not allow them.

Comment (kchilton2) [reply extras/rya.streams/query-manager/src/main/config/configuration.xml:47]: Maybe we should put a restriction on value in the schema though. It should only be a positive, non-zero, integer.

Comment (isper3at) [reply extras/rya.streams/query-manager/src/main/config/configuration.xml:47]: yeah agreed

**Review by isper3at — APPROVED** [review]

**Comment by isper3at** [comment]
:shipit:

**Pull request merged**